templates/pages/account/orders/invoice.html

```diff
@@ -56,6 +56,23 @@ <h3>{{lang 'account.orders.details.ship_to'}}</h3>
                     </div>
                 {{/unless}}
             {{/if}}
+            {{#if order.pickup_address}}
+                {{#unless order.has_multiple_shipping_addresses}}
```

Comment:
`order.has_multiple_shipping_addresses` means order can have 1 pickup and 1 shipping right?

Reply:
yep I used the same variable name for backwards compatibility

Reply:
👍